doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,33 @@
-=======================
-Release Build Creation
-=======================
+=========================================================
+Compile & Build Complete Cortx Stack Via Docker Container
+=========================================================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Prerequisites to setup Virtual Machine
+*****************************************
+
+- Single-Node VM deployment:
+  
+  - Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system with the following configuration in Virtual Machine (VM):
+  
+    - RAM: 4GB
+    - CPU: 1 core
+    - Total Disks: 2 (Raw disks)
```

Comment:
I have only ever tested with a minimum of 3 disks. I think you need 2 extra disks and 1 main disk? @shailesh-vaidya what do you think?

Reply:
yes corrected as tested

Reply:
but I tested with 20G so it works

Reply:
updated 50GB